css-transitions/styles.css

```diff
@@ -0,0 +1,54 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: sans-serif;
+  background-color: rgb(245 245 245);
+}
+
+section {
+  max-width: 992px;
+  margin: 2rem auto;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.custom-button {
+  padding: 0.5rem 1.5rem;
+  border: none;
+  border-radius: 0.25rem;
+  font-size: 1rem;
+  color: white;
+  background-color: blue;
+  transition: 0.5s;
```

Comment:
Be sure to specify which CSS properties you want to transition. the default is `all`, which can be very computationally expensive.

https://developer.mozilla.org/en-US/docs/Web/CSS/transition#syntax